.gitignore

```diff
@@ -1,3 +1,4 @@
 inagoctl
 .gobuild/
 coverage.txt
+profile.out
```

Comment:
from: https://github.com/giantswarm/inago/blob/master/go.test.sh#L10
If the test panics the profile.out is not deleted, which might be fine, but I wouldn't want to commit it.